src/modules/Api.ts

```diff
@@ -0,0 +1,186 @@
+import request from 'request-promise-native';
+import BigNumber from 'bignumber.js';
+import queryString from 'query-string';
+import {
+  LimitOrder,
+  address,
+  Integer,
+  SigningMethod,
+  ApiOrder,
+  ApiAccount,
+} from '../types';
+import { LimitOrders } from './LimitOrders';
+
+const FOUR_WEEKS_IN_SECONDS = 60 * 60 * 24 * 28;
+const TAKER_ACCOUNT_OWNER = '0xf809e07870dca762B9536d61A4fBEF1a17178092';
```

Comment:
nit: ENV variable

Reply:
Solo is not set up to use ENV variables right now because it's a client library

Reply:
@AntonioJuliano Well there should probably be some configuration outside of code. An ENV variable would work fine, it's just baked into the code on compile.